content/TrueCommand/Administration/Settings/SAML.md

```diff
@@ -193,116 +190,111 @@ This procedure assumes the Windows administrator user is QE. Substitute your sys
 
    i. Click **Finish**, **Apply**, and **OK**.
 
-8. Close Active Directory.
-
-9. Go to the TrueCommand login page and use the **SAML Login**.
+8. Close **Active Directory**.
 
+9. Go to the TrueCommand login page and use the SAML Login.
```

Comment:
SAML Login is the UI element so it should be in bold, suggest changing to:

9. Go to the TrueCommand login page and use the **SAML Login**.

Reply:
Fixed